src/styles/blocks/card.css

```diff
@@ -0,0 +1,42 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__imac-image {
+  width: 160px;
+  height: 134px;
+
+  margin-top: 32px;
+}
```

Comment:
i would imagine a shop having multiple products. maybe it'll be better to name it `.card__image`